app/assets/javascripts/controllers/automation_manager/automation_manager_form_controller.js

```diff
@@ -51,7 +50,7 @@ ManageIQ.angular.app.controller('automationManagerFormController', ['$http', '$s
     vm.automationManagerModel.name = data.name;
     vm.automationManagerModel.zone = data.zone;
     vm.automationManagerModel.url = data.url;
-    vm.automationManagerModel.verify_ssl = data.verify_ssl == "1";
+    vm.automationManagerModel.verify_ssl = data.verify_ssl === '1';
```

Comment:
While this is a good thing to do, it's best to keep the changes in the PR only specific to the PR topic.

Reply:
I am very sorry for this but when i see something like this my eyes can't rest until i chnage it :D.